plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/PassthroughQueryPageSource.java

```diff
@@ -41,40 +53,55 @@ public PassthroughQueryPageSource(ElasticsearchClient client, ElasticsearchTable
         long start = System.nanoTime();
         result = client.executeQuery(table.getIndex(), table.getQuery().get());
         readTimeNanos = System.nanoTime() - start;
+
+        if (result.v2()) {
```

Comment:
Can we name this and to something meaningful?

I think it's some sort of success flag.